components/clp-py-utils/clp_py_utils/clp_config.py

```
@@ -605,6 +627,32 @@ def validate_port(cls, field):
         return field
 
 
+class JobFrequency(BaseModel):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)